src/slack.js

```diff
@@ -281,84 +269,72 @@ async function handleCSVUpload(event, bot, say) {
-            },
-            action_id: ACTION_ID_SEND_SK,
-            value: 'SK',
-            style: 'primary',
-            confirm: {
-              title: {
-                type: 'plain_text',
-                text: 'Are you sure?',
-              },
-              text: {
-                type: 'plain_text',
-                text: 'Do you really want to send these Slovak invoices?',
-              },
+          Object.assign(
+            getBlock('plain_text', `Send ${invoices.length} invoices`, 'button'),
+            {
+              action_id: ACTION_ID_SEND_SK,
+              value: 'SK',
+              style: 'primary',
               confirm: {
-                type: 'plain_text',
-                text: 'Yes, send them all',
-              },
-              deny: {
-                type: 'plain_text',
-                text: 'No',
-              },
-            },
-          },
-          {
-            type: 'button',
-            text: {
-              type: 'plain_text',
-              text: `Send ${invoices.length} invoices (EN)`,
-            },
-            action_id: ACTION_ID_SEND_EN,
-            value: 'EN',
-            confirm: {
-              title: {
-                type: 'plain_text',
-                text: 'Are you sure?',
-              },
-              text: {
-                type: 'plain_text',
-                text: 'Do you really want to send these English invoices?',
+                title: {
+                  type: 'plain_text',
+                  text: 'Are you sure?',
+                },
+                text: {
+                  type: 'plain_text',
+                  text: 'Do you really want to send these Slovak invoices?',
+                },
+                confirm: {
+                  type: 'plain_text',
+                  text: 'Yes, send them all',
+                },
+                deny: {
+                  type: 'plain_text',
+                  text: 'No',
+                },
               },
+            }
+          ),
+          Object.assign(
+            getBlock('plain_text', `Send ${invoices.length} invoices (EN)`, 'button'),
+            {
+              action_id: ACTION_ID_SEND_EN,
+              value: 'EN',
               confirm: {
-                type: 'plain_text',
-                text: 'Yes, send them all',
-              },
-              deny: {
-                type: 'plain_text',
-                text: 'No',
+                title: {
+                  type: 'plain_text',
+                  text: 'Are you sure?',
+                },
+                text: {
+                  type: 'plain_text',
+                  text: 'Do you really want to send these English invoices?',
+                },
+                confirm: {
+                  type: 'plain_text',
+                  text: 'Yes, send them all',
+                },
+                deny: {
+                  type: 'plain_text',
+                  text: 'No',
+                },
               },
```

Comment:
in the task, I wrote: "refactor blocks - extract and reuse". somehow I'd like to collapse this block, so I actually lean towards extracting the whole action objects out to a new `constants`, `slackConstants`, or `blocks` file. also, as I mentioned in another comment, we could have a `getActionBlock` function that will somehow compose this. or better, specific `getSendInvoicesActionBlock`, because the SK and EN buttons are more or less the same

Reply:
originally I thought that extracting block creation into `getBlock` function will be sufficent, but I agree that moving that "logic" into separate file looks much more convenient